METRICS.md

```diff
@@ -59,7 +60,6 @@ We collect data for the following extension events:
 
 - When panel navigation arrow icons are clicked
 
-
 ### In-page events:
 
 - When the Relay icon is injected into an email input
```

Comment:
Wow good catch. I'd like to get this part merged so we can test it ASAP, since the website more directly supports the Firefox integration, which is our current priority. I filed https://mozilla-hub.atlassian.net/browse/MPP-3831 as a follow-up to update the metrics_event endpoint for GA4.